### PR TITLE
Build and push collections

### PR DESCRIPTION
**Pull request by sabre1041**
Build and publish sample collections

### CODE REVIEW

File: .github/workflows/build-push-collections.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
```
Comment (usrbinkat) [inline .github/workflows/build-push-collections.yml]: Are we using qemu / buildx in later steps of this action?

Comment (sabre1041) [reply .github/workflows/build-push-collections.yml]: supports registry login

Comment (usrbinkat) [reply .github/workflows/build-push-collections.yml]: is the qemu step required for login?

Comment (sabre1041) [reply .github/workflows/build-push-collections.yml]: Removed!

**Review by afflom — APPROVED** [review]

**Review by jpower432 — APPROVED** [review]

**Pull request merged**